README.md

```diff
@@ -95,12 +95,23 @@ the instructions in [podman.md](podman.md).
 To run the tests, type `./gradlew test` from the `hibernate-reactive` 
 directory.
 
+#### Starting databases automatically with Testcontainers
+
+If you have Docker installed, the container instances will be automatically started
+by running the tests if you select the `-Pdocker` profile like so:
```

Comment:
I think it's more consistent with the previous paragraph (and shorter) if you you rewrite it to something like: 
```
#### If you have Docker

The build will start the container automatically if you select the profile `-Pdocker`:
```

Reply:
I guess the profile is `docker`  (and not `-Pdocker`).